PostNewQuestions.php

```diff
@@ -144,3 +150,7 @@ public function getNewestPostsInStackOverflow(string $tag): ?array
     $postData = $newPostService->convertQuestionToSlackData($tag, $newestQuestions);
     $newPostService->sendPostToSlack($tag, $postData);
 }
+
+if (!empty($postData)) {
```

Comment:
Why? If there were no questions, there is no need to look for those "non-questions" again… Or is that a safeguard of some sort?

Reply:
Yes, I think this is a kind of safeguard. I think the check doesn't hurt as it shouldn't make the subsequent API call more expensive, but at the same time the script doesn't trust the API 100% that there is no message if it doesn't return a message. Maybe Spoonerweb has some experience there :)